File: inoseask_app/app/src/main/java/com/example/sampleapplication/MainActivity.java
package com.example.sampleapplication;

import android.Manifest;
import android.content.Intent;
import android.content.pm.PackageManager;
import android.net.Uri;
import android.os.Bundle;
import android.widget.Toast;
import androidx.appcompat.app.AppCompatActivity;



public class MainActivity extends AppCompatActivity {
    private static final int REQUEST_PERMISSION = 999;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        init();
    }

    private void init() {
        //String urlScheme ="https://ibk.kr/mpd?p=ava";
        //String urlScheme ="ibkmportal://ibk?p=ava";
        String urlScheme ="kakao816961d76321eb6c24be53904a5d4ca9://kakaolink";
        Intent intent = new Intent();
        intent.setAction(Intent.ACTION_VIEW);
        intent.setData(Uri.parse(urlScheme));
        startActivity(intent);

        finishAffinity();
    }

    @Override
    public void onRequestPermissionsResult(int requestCode, String[] permissions, int[] grantResults) {
        super.onRequestPermissionsResult(requestCode, permissions, grantResults);
    }


    @Override
    public void onBackPressed() {
        super.onBackPressed();
    }
//
//
//        intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK);
//        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
}

File: inoseask_app/app/src/main/java/com/example/sampleapplication/MainActivity.kt
package com.example.sampleapplication

import android.app.Activity
import android.content.Intent
import android.net.Uri
import android.os.Bundle
import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        installSplashScreen()
        setContentView(R.layout.activity_main)
        init()
    }

    private fun init() {
        //String urlScheme ="https://ibk.kr/mpd?p=ava";
        //String urlScheme ="ibkmportal://ibk?p=ava";
        val urlScheme = "kakao816961d76321eb6c24be53904a5d4ca9://kakaolink"
        val intent = Intent()
        intent.action = Intent.ACTION_VIEW
        intent.data = Uri.parse(urlScheme)
        startActivity(intent)
        finishAffinity()
    }
//
//
//        intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TASK);
//        intent.addFlags(Intent.FLAG_ACTIVITY_NEW_TASK);
}